src/docs/reference/pricing.md

```diff
@@ -2,37 +2,225 @@
+## Add-on services
 
-[You can see the offerings of each tier on this page.](/reference/plans)
+Railway offers [Business Class Support](https://www.notion.so/5c47cc42b2cb456c85d125b512bb8763?pvs=21) as an add-on service for teams on the Pro plan (Business Class Support is included with Enterprise). [Contact us](mailto:contact@railway.app) to get started.
 
-## Pricing Rates
+## Credits
 
-Railway computes the usage cost with the following formula:
+Credits are available on Railway as part of the Trial plan as well as a payment method for users who prefer to prepay for their subscription and usage on Railway.
 
-`Num Services` x (`Service Time` (in minutes) x (`Memory usage` (in GB)) + `vCPU usage` (in cores))
+### One-time grant of credits on the Trial plan
 
-And our rates for usage are as follows:
+Users who create a Trial new account receive a free one-time grant of $5. Railway will expend any free credit before consuming any paid credits.
 
-**Memory** - $10 / GB per month ($0.000231 / GB / minute)
+### Credits as a payment method on the Hobby plan
 
-**vCPU** - $20 / vCPU per month ($0.000463 / vCPU / minute)
+On Railway, you can pay for your Hobby plan subscription and any additional usage with pre-purchased credits. This results in some specific behavior.
 
-Keep in mind, if your project has PR deploys enabled, Railway deploys a mirror copy of your service(s) based on the environment it forks from (`production` by default). You are billed for the services that are spun while the ephemeral environment is active.
+- If your usage within a billing period exceeds your credit balance, you will no longer be able to deploy, and your projects will be paused
+- If you cancel your Hobby plan subscription, any remaining credit balance will be lost
 
 ## Billing Management
 
-You can add and manage your payment info under the billing page and see historical billing usage under the account billing page: [railway.app/account/billing](https://railway.app/account/billing)
+You can add and manage your payment info under the billing page and see historical billing usage under the account billing page: [railway.app/account/billing](https://railway.app/account/billing)
+
+Within this page, you can also top-up on credits for your account if you prefer to pay for the Hobby plan in a prepaid fashion.
+
+For more information, read the [Billing](https://docs.railway.app/reference/pricing#billing-management) docs.
+
+## Verification
+
+Verification is a fraud prevention measure designed to filter out spammy actors while incentivizing legitimate actors.
+
+Verification status depends on a number of factors, including the age and activity of your GitHub account.
+
+If you have trouble getting verified (often because your GitHub account is not up to the verification standard), don’t fret! You are still able to use a credit card and purchase Railway plans.
```

Comment:
"... and purchase Railway plans" -- "Railway plans" should ideally link to `/reference/pricing#plans`